Mathlib/Algebra/Polynomial/Eval/Degree.lean

```diff
@@ -127,6 +127,10 @@ def mapEquiv (e : R ≃+* S) : R[X] ≃+* S[X] :=
   RingEquiv.ofHomInv (mapRingHom (e : R →+* S)) (mapRingHom (e.symm : S →+* R)) (by ext; simp)
     (by ext; simp)
 
+theorem irreducible_mapEquiv_iff (e : R ≃+* S) (p : R[X]) :
+    Irreducible (Polynomial.mapEquiv e p) ↔ Irreducible p :=
+  MulEquiv.irreducible_iff (Polynomial.mapEquiv e)
```

Comment:
Do we need this? Should we have it for every MulEquivClass we define?

Reply:
Yes, I am not sure about this. This proof could easily be inlined where it is needed instead.

Reply:
Ok, I finally to remove this lemma and inline the proof.

Thanks for the review!